assets/styles/docs.scss

```diff
@@ -142,6 +142,32 @@ $bd-gutter-x: 3rem;
     grid-area: toc;
 }
 
+.docs {
+    .bd-toc {
+        grid-area: sidebar;
+
+        @media (max-width: 992px) {
+            display: none;
+        }
+    }
+    .bd-content {
+        @media (max-width: 1110px) and (min-width: 992px) {
+            max-width: 88%;
+            margin: 0;
+        }
+    }
+    .bd-main {
+        .bd-toc {
+            display: none !important;
+
+            @media (max-width: 992px) {
+                display: block !important;
+                grid-area: toc !important;
+            }
```

Comment:
Do we need `!important`? I don't see any display overrides that would prevent it from rendering as required.